character/create.html

```diff
@@ -125,6 +125,172 @@ <h2>General Information</h2>
+                                        <th scope="row">Reactions</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Brains</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Discipline</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Toughness</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Brawn</th>
+                                        <td>-</td>
+                                    </tr>
+                                </tbody>
+                            </table>
+                        </div>
+                        <div class="col-auto table-responsive">
+                            <table class="table table-bordered">
+                                <thead class="table-light">
+                                    <th scope="col">3<sup>rd</sup> Sphere</th>
+                                    <th scope="col">Bonus Dice</th>
+                                </thead>
+                                <tbody>
+                                    <tr>
+                                        <th scope="row">Operations</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Science</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Mental</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Leadership</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Brute</th>
+                                        <td>-</td>
+                                    </tr>
+                                    <tr>
+                                        <th scope="row">Physical</th>
+                                        <td>-</td>
+                                    </tr>
+                                </tbody>
+                            </table>
+
+                            <!-- Pad all of the table rows to the height of the single tallest row for consistency -->

```

Comment:
This is better achieved with CSS, but I can take care of that at a later time. The method is fine, it's just a bit roundabout to do it via javascript rather than updating the css directly.